### ISSUE TITLE
sb3 player need a progress bar just like sb2 player

### ISSUE THREAD

**Issue by cryanis** [LLK/scratch-gui#2942]
Some items have to be loaded for a long time.

**Comment by towerofnix** [comment]
This is probably already being tracked in the issue #1852.

**Comment by cryanis** [comment]
@towerofnix yes, the standalone sb3 player need a progress bar and the user experience will be better. But it doesn't seem to exist now.
